src/index.js

```diff
@@ -206,12 +205,11 @@ class Service {
   update(id, data) {
     return this._get(id).then(getData => {
       data.id = id;
-
       return this.table.get(getData.id)
         .replace(data, {
           returnChanges: true
         }).run()
-        .then(result => result.changes[0].new_val);
+        .then(result => (result.changes && result.changes.length) ? result.changes[0].new_val : {});
```

Comment:
I think we probably want to return `data` instead of `{}`. Some client libraries will remove attributes missing from the response.